sound/soc/sof/sof-priv.h

```diff
@@ -582,7 +587,8 @@ void sof_print_oops_and_stack(struct snd_sof_dev *sdev, const char *level,
 			      u32 panic_code, u32 tracep_code, void *oops,
 			      struct sof_ipc_panic_info *panic_info,
 			      void *stack, size_t stack_words);
-int snd_sof_init_trace_ipc(struct snd_sof_dev *sdev);
+int snd_sof_trace_suspend(struct snd_sof_dev *sdev, int pm_state);
```

Comment:
I think to align with the SOF client interface, this should be:
```
int sof_suspend_clients(struct snd_sof_dev *sdev, pm_message_t pm_state);
```

Reply:
Yes, you need to pass the `pm_message_t`